test/unit/lib/is-object.js

```diff
@@ -0,0 +1,140 @@
+const { d, nextInt, testRunner, tquire } = deps;
```

Comment:
This file has some code comments that I should pull out. I'm struggling to remember what they were even for, because I wrote them months ago.